common/views/themes/base/container.ts

```diff
@@ -65,14 +65,4 @@ export const container = `
   `
   )}
 }
-
-.container--scroll-cream {
```

Comment:
Was only used once (`content/webapp/pages/stories.tsx`) so I moved it there directly.